### PR TITLE
Dimethyl tutorial

### PR DESCRIPTION
**Pull request by GeorgWa**
Adds a step by step tutorial for performing transfer learning with the alphaDIA GUI.

### CODE REVIEW

**Comment by lucas-diedrich** [comment]
[Library-free DIA search using the GUI - alphadia documentation.pdf](https://github.com/user-attachments/files/17380463/Library-free.DIA.search.using.the.GUI.-.alphadia.documentation.pdf)

Tutorial as `.pdf`

**Comment by lucas-diedrich** [comment]
[DIA Transfer Learning for Dimethyl Labeled Samples - alphadia documentation.pdf](https://github.com/user-attachments/files/17380511/DIA.Transfer.Learning.for.Dimethyl.Labeled.Samples.-.alphadia.documentation.pdf)

Tutorial as `.pdf`

**Review by lucas-diedrich — APPROVED** [review]
Followed the tutorial through, everything was very clear! The only nit-picky thing might be to add a note to disable the Transfer Library/Transfer Learning settings after the initial step.

File: docs/guides/transfer-dimethyl.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+<img src="../_static/images/transfer-dimethyl/second_settings.png" width="100%" height="auto">
+Start the second search as before. This search will be significantly faster than the two before.
+
+## 6. Results
+After completing both searches, you'll find full search outputs in both the `second_pass` folder. For downstream analysis, use the precursor-level file `precursors.tsv` or the protein matrix in `pg.matrix.tsv`. If you want to use the PeptDeep models, you can find them in `transfer_pass/peptdeep.transfer`.
+
+You can get a quick overview from the contents of the `stat.tsv` file in the `second_pass` folder. This will show you the number of precursors and protein groups identified across your samples, along with other relevant metrics.
+
+Alltough the different parameters do not allow for fair comparison of absolut numbers between steps, we can still get an idea how transfer learning affects the number of identified precursors.
```
Comment (vbrennsteiner) [inline docs/guides/transfer-dimethyl.md:121]: nit: 'Although'

File: docs/index.md [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+
+| Platform              | Empirical lib | Predicted lib |
+| :---------------- | :------: | :----: |
+| Thermo .raw |   ✅   | ✅ |
+| Sciex .wiff |   ✅   | ✅ |
+| Bruker .d |  ✅   | ⚠️ |
+
+:::{admonition} Predicted libraries with Bruker .d data
+:class: warning
+Alltough search is possible, alphaDIA's feature-free search takes a long time with fully predicted libraries. We are still evaluating how to better support fully predicted libraries.
```
Comment (vbrennsteiner) [inline docs/index.md:32]: nit: Although

**Review by vbrennsteiner — COMMENTED** [review]
No major issues found, just nit comments on one image not showing up in the GitHub preview and two typos.

**Pull request merged**